index.html

```diff
@@ -27,33 +27,292 @@ <h2>I’am a Software Developer</h2>
+                                <a href="#">bootstrap</a>
+                            </li>
+                            <li>
+                                <a href="#">Ruby</a>
+                            </li>
+                        </ul>
+                    </div>
+                    <button type="submit">See Project</button>
+
+                </div>
+            </div>
+            <div class="project-background">
+                <div class="project-foreground">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <div class="project-keyword">
+                        <ul>
+
+                            <li>
+                                <a href="#">html</a>
+                            </li>
+                            <li>
+                                <a href="#">bootstrap</a>
+                            </li>
+                            <li>
+                                <a href="#">Ruby</a>
+                            </li>
+                        </ul>
+                    </div>
+                    <button type="submit">See Project</button>
+
+                </div>
+            </div>
+            <div class="project-background">
+                <div class="project-foreground">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <div class="project-keyword">
+                        <ul>
+
+                            <li>
+                                <a href="#">html</a>
+                            </li>
+                            <li>
+                                <a href="#">bootstrap</a>
+                            </li>
+                            <li>
+                                <a href="#">Ruby</a>
+                            </li>
+                        </ul>
+                    </div>
+                    <button type="submit">See Project</button>
+
+                </div>
+            </div>
+
+        </div>
+
+    </section>
+    <section>
+        <div class="about">
+            <div class="about-header">
+                <h2>About me</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
+                    of
+                    my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+                    to
+                    contact me.</p>
+                <button type="submit">Get My Resume</button>
+
+            </div>
+            <div class="about-img">
+                <img src="./images/about-me-img.png" alt="background img">
+            </div>
```

Comment:
- [ ] for aesthetic reasons backgrounds images should be implemented using CSS background-image and not image tags, kindly fix it 🙏 [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

Reply:
@JakariaJishan i didn't understand sorry can you tell me what you mean exactly?